### PR TITLE
Add Pixel 6 support

### PR DESCRIPTION
**Pull request by dbaxa**
Signed-off-by: David Black <dblack@atlassian.com>

### CODE REVIEW

**Comment by dbaxa** [comment]
This seems to work, but I may have well missed something.

File: app/src/main/java/app/attestation/auditor/AttestationProtocol.java [hunk 1]
```diff
@@ -489,6 +490,8 @@
                     new DeviceInfo(R.string.device_pixel_5_generic, 4, 41, false /* uses new API */, true, R.string.os_stock))
             .put("1DD694CE00BF131AD61CEB576B7DCC41CF7F9B2C418F4C12B2B8F3E9A1EA911D",
                     new DeviceInfo(R.string.device_pixel_5a, 4, 41, false /* uses new API */, true, R.string.os_stock))
+            .put("0F6E75C80183B5DEC074B0054D4271E99389EBE4B136B0819DE1F150BA0FF9D7",
```
Comment (dbaxa) [inline app/src/main/java/app/attestation/auditor/AttestationProtocol.java:493]: Hopefully this matches up to the sample data that I have submitted for the device.

**Comment by dbaxa** [comment]
cc @thestinger .

**Comment by thestinger** [comment]
It also needs to be added to the list of devices where StrongBox is used and an entry added to the StrongBox table too.

**Comment by thestinger** [comment]
This is implemented now along with Pixel 6 Pro support. It's available in v35.

**Pull request closed without merging**